apps/devhub/tests/test_views.py

```diff
@@ -1207,6 +1212,23 @@ def test_delete_and_recreate_credentials(self):
         assert new_key.key != old_key.key
         assert new_key.secret != old_key.secret
 
+    def test_retreive_credentials(self):
+        client = Client(enforce_csrf_checks=True)
+        assert client.login(username='regular@mozilla.com', password='password')
+
+        response = client.get(reverse('devhub.api_creds'),
+                                   {'key': 'abc123'})
```

Comment:
because key is defined in another function you should reference it by variable like `'key': self.api_key.key`